src/Explorer/Controls/NotebookGallery/GalleryViewerComponent.tsx

```diff
@@ -410,6 +432,8 @@ export class GalleryViewerComponent extends React.Component<GalleryViewerCompone
           onRenderCell={this.onRenderCell}
         />
       </FocusZone>
+    ) : (
+      <Spinner size={SpinnerSize.large} />
```

Comment:
Is this needed?

Reply:
this takes care of public gallery and official samples.